CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.12.0)
 
-project(ni_grpc_device_server C CXX)
+project(ni_grpc_device_server C CXX
+  VERSION 1.5.2)
```

Comment:
This populates the `PROJECT_VERSION_*` variables as per the CMake docs [here](https://cmake.org/cmake/help/latest/command/project.html).